.github/workflows/pr-checks.yml

```
@@ -0,0 +1,464 @@
+      pull-requests: write  # Needed to comment on PRs
+      issues: write         # Needed to create/update issue comments
+      actions: read        # Needed to upload artifacts
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Validate Gradle wrapper
+      uses: gradle/wrapper-validation-action@v1
```

actionlint (1.7.7)

the runner of "gradle/wrapper-validation-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)